### ISSUE TITLE
How does this work? Reject all by default or accept all by default?

### ISSUE THREAD

**Issue by icougil** [OhMyGuus/I-Dont-Care-About-Cookies#94]
Hi!

I found really interesting the idea of this extension, but at the same time I'd like to know how it works. 

I mean, I guess it is going to accept by default all the cookies in all the "compatible" websites, right? I'd like to do it in the other way around: I don't want to accept any cookie by default and reject all of them if it's possible, and if not, just leave the popup opened. 

I guess this is not the default behaviour of this extension, right?

Best,

**Comment by OhMyGuus** [comment]
At the moment it uses in most cases it just uses CSS/JavaScript to hide the pop-up/warning. This doesn't accept or deny the cookies and leaves the website to decide what to do. However, any good website will see this as a denial of cookies. In some cases, it clicks the pop-up or set a cookie to accept the required cookies for a functional website, but only if hiding using css/request blocking doesn't work.
A recommendation as well is to use an extension like "Cookie autodelete" to get the best of two worlds.


TLDR: It does everything to hide/remove the popup, it doesn't reject it if it's possible at the moment.

**Comment by icougil** [comment]
Got it, thank you! 👍 
Then as it states, everything it is focused on a CSS / GUI mechanisms to hide the corresponding popup's without rejecting/accepting any cookie.
PS: I tried with some Spanish online newspapers and it is working like a charm 👌
